ui/ts/models/log.ts

```diff
@@ -58,39 +58,58 @@ module Models {
         true
       );
 
-      private _url(): string {
-        let url: string = "/_status/logs/";
+      /**
+       * buildURL creates the route and query parameters based on the current
+       * state of all the properties on entries.
+       * TODO(bram): consider exporting this to a utility function and maybe
+       * adding an interface for these top level models. Tamir also suggested
+       * moving to a single param with a tighter serialized format.
+       */
+      buildURL(): string {
+        let url: string = "/logs/";
```

Comment:
okay, I found something better for this: use [m.route.buildQueryString](http://lhorie.github.io/mithril/mithril.route.html#buildQueryString).  

If you want to go all the way, you could pull in [URI.js](http://medialize.github.io/URI.js/), which appears to be a stable, complete and basically standard library for manipulating URIs.  Mithril's builtin URI functions are based on it, apparently.